src/network/dandelion.py

```diff
@@ -47,15 +49,31 @@ def poissonTimeout(start=None, average=0):
             average = FLUFF_TRIGGER_MEAN_DELAY
         return start + expovariate(1.0 / average) + FLUFF_TRIGGER_FIXED_DELAY
 
+    def init_pool(self, pool):
+        """pass pool instance"""
+        self.pool = pool
+
+    def is_dandelion_enabled(self, config):
```

Comment:
I would call this `init_dandelion_enabled`

Reply:
done